native-to-erc20/oracle/.env.example

```diff
@@ -28,6 +27,8 @@ FOREIGN_GAS_PRICE_FACTOR=0.1
 FOREIGN_GAS_PRICE_UPDATE_INTERVAL=600000
 FOREIGN_DEFAULT_GAS_PRICE=50000000000
 
+FOREIGN_VALIDATOR_RESPONSIBLE_TO_RELAY=false
```

Comment:
Where is this used?

Reply:
thank for noticing!.. fixed, see the latest commit